x/airdrop/keeper/grpc_query_test.go

```diff
@@ -118,32 +117,28 @@ func (s *KeeperGrpcQueryTestSuite) TestQueryAllFund() {
 		DripAmount: sdk.NewInt(40),
 	}
 
-	var (
-		req         *types.QueryAllFundsRequest
-		expResponse types.QueryAllFundsResponse
-	)
-
 	testCases := []struct {
 		msg      string
-		malleate func()
+		testFunc func() (*types.QueryAllFundsRequest, types.QueryAllFundsResponse)
```

Comment:
why not returning a `*types.QueryAllFundsResponse` here ?

Reply:
Well spotted, I didn't meant to implement it in that way